src/resources/js/tickets.js

```diff
@@ -471,6 +473,22 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 				}
 			} );
 
```

Comment:
If `start date > end date`, what happens? Also, when `end date < start date`, what happens? In those situations, I think we should update to `start date = end date`, like our ticket sales dates do.

Reply:
Yes, we do have `start date > end date` covered in the test: https://github.com/the-events-calendar/event-tickets/blob/7ef54fb5114bbb1a7b06a8912b552066bd9c4a6e/tests/commerce_integration/TEC/Tickets/Commerce/TicketTest.php#L161

Your second logic is basically the same as the first one, so that also is covered by the same test. Feel free to look into the test cases and let me know if I have missed something.

From the editor we do not allow to save those values when the range is not valid, but even if they are added it should handle properly according the added tests.